docs/topics/auth/default.txt

```diff
@@ -1290,6 +1290,12 @@ implementation details see :ref:`using-the-views`.
         The ``current_app`` parameter is deprecated and will be removed in
         Django 2.0. Callers should set ``request.current_app`` instead.
 
+    .. versionadded:: 1.9
+
+        The ``extra_email_context`` parameter is added which would be merged
```

Comment:
The parameter should be documented as the other optional arguments are (in the bulleted list above) with this note remaining here simply saying "The extra_email_context parameter was added." The idea is to make the note "self-contained" [as described here](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features).